Update/some_007.txt

```diff
@@ -2008,7 +2008,7 @@ void main()
 //r私は、…素直には喜べなかった。だって私は、…こんなにも優しい山瀬さんに対して、ひどい嘘をついてしまっている…。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "私は、…素直には喜べなかった。だって私は、…こんなにも優しい山瀬さんに対して、ひどい嘘をついてしまっている…。",
-			NULL, "I... honestly, I couldn't rejoice. Because I... have been lying to a person as kind as Yamase-san...", Line_Normal);
+			NULL, "I... couldn't bring myself to genuinely rejoice. Because I... hde been lying to someone as kind as her...", Line_Normal);
```

Comment:
```suggestion
			NULL, "I... couldn't bring myself to genuinely rejoice. Because I... had been lying to someone as kind as her...", Line_Normal);
```